assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/index.test.js

```diff
@@ -498,7 +543,51 @@ describe( 'AudienceTilesWidget', () => {
 			)
 		).toBeInTheDocument();
 
-		await act( () => waitForTimeouts( 100 ) );
+		await act( () => waitForTimeouts( 150 ) );
```

Comment:
This test was still proving to be a bit flaky, hence increasing the timeout which was added to improve stability.